src/utilities/modules/vaccines.js

```diff
@@ -82,6 +82,70 @@ export const extractBreaches = ({ sensorLogs = [], database }) => {
   );
 };
 
+/**
+ * Helper method for sensorLogExtractBreaches. Extracts
+ * statistics for a set of sensorlogs and itembatches.
+ * @param {Realm.results<ItemBatch>} itemBatches
+ * @param {Realm.results<SensorLog>} sensorLogs
+ */
+const extractBreachStatistics = (itemBatches, sensorLogs) => ({
+  location: sensorLogs.location,
+  numberOfAffectedBatches: itemBatches.length,
+  affectedQuantity: itemBatches.sum('numberOfPacks'),
+  exposureRange: {
+    minTemperature: itemBatches.min('temperature') || Infinity,
+    maxTemperature: itemBatches.max('temperautre') || -Infinity,
+  },
+  breachDuration: {
+    startDate: itemBatches.max('timestamp') || null,
+    endDate: itemBatches.min('timestamp') || null,
```

Comment:
sensorLogs.max/min, for both exposureRange and breachDuration

Reply:
89e3f18862158caacbebe4e4f2ce61fdba90047c